src/passes/HeapSnapshotAnalysis.cpp

```diff
@@ -117,6 +117,44 @@ struct HeapSnapshotAnalysis : Pass {
       auto [name, allocs] = topTypesByAllocations[i];
       std::cout << "  " << name << ": " << allocs << " objects\n";
     }
+
+    // Measure the savings of removing vtable pointers. Assume any object that
+    // contains an edge to another object that contains an edge to a "system /
+    // WasmFuncRef" contains a vtable pointer.
```

Comment:
How precise do you think this is?

Reply:
I _think_ it should be fairly precise. I don't know of any other patterns involving function references, do you? I have samples with name sections so I can do another analysis where I just look for "vtable" in the type name and compare the results to make sure.

Reply:
I can't say I know of one, but I remember talk about caching function pointers in some cases... Good idea to compare to types with "vtable" in the name, yeah, that would give a lot more confidence.